src/components/heading/heading.scss

```
@@ -64,12 +65,13 @@ $max-width-permalink-shift: math.div($min-width-permalink-shift * 16 - 1, 16);
 /**
  * Weight modifier
  *
- * The default font weight for headings is set to 700. This modifier will
- * change that to a lighter weight of 300.
+ * The default font weight for headings is set to font-weight.$bold.
```

```suggestion
 * The default font weight for headings is normally defined by heading level.
```

(The largest heading levels are actually bolder than bold!)